gallery/visualtests/nxTabs.js

```diff
@@ -62,6 +63,16 @@ describe('NxTabs', function() {
             `${tabSimpleExampleSelector} .nx-tab:nth-child(2)`));
   });
 
+  it('shows tooltip when text is truncated', async function() {
```

Comment:
See the third param to `hoverTest` in `testUtils`.  I believe it could simplify this test substantially.

Reply:
> See the third param to `hoverTest` in `testUtils`. I believe it could simplify this test substantially.

Simplified here:  d9f6814fe4fdadf3b4d3d2b2adac68746afb588e